src/js/components/search/filters/timePeriod/TimePeriod.jsx

```diff
@@ -82,9 +98,28 @@ export default class TimePeriod extends React.Component {
         }
     }
 
+    saveSelected(arrayFY, allSelected) {
+        this.setState({
+            selectedFY: arrayFY,
+            allFY: allSelected
+        });
+    }
+
+    saveAll(arrayFY, allFY) {
```

Comment:
Do we need this function? Could we get rid of it and use `saveSelected` instead?